tests/steps/interfaces.ts

```diff
@@ -10,4 +11,8 @@ When("I try to get all GPS interface data", async function () {
   this.lastResponse = await api.get(GPS, this.config, false)
 });
 
+When("I get all AISShip interface data", async function () {
+  this.lastResponse = await api.get(AISShips, this.config, false)
```

Comment:
Please remove the `false` argument at the end of this method. When we explicitly input `false`, it's saying that this method should continue running even if the GET request fails. 

We have two distinct methods for these cases (as you can infer from GPS). The `I get all GPS interface data` step is used for requests where we expect the request to succeed, and `I try to get all GPS interface data` is used for requests where the request is expected to fail.